execution_engine/src/core/engine_state/genesis.rs

```diff
@@ -68,6 +68,7 @@ use crate::{
         protocol_data::ProtocolData,
     },
 };
+use casper_types::contracts::ContractPackageStatus;
```

Comment:
This one slipped in last review - merge imports (I know that feel I usally trust my IDE too much)